### ISSUE TITLE
Sketchup::Drawingelement#visible= incorrect parameter and return value.

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/api-issue-tracker#565]
#### SketchUp Ruby API Documentation Issue

**[`Sketchup::Drawingelement#visible=`](https://ruby.sketchup.com/Sketchup/Drawingelement.html#visible=-instance_method) method documentation is incorrect for the parameter and return value.**

The following appears to have been pasted from the [`#hidden=`]() method and is **contrary** to how the [`#visible=`](https://ruby.sketchup.com/Sketchup/Drawingelement.html#visible=-instance_method) method works.

>Parameters:
> 
> * visibility (Boolean) — true if you want to hide the element, false if not
>
> Returns:
> 
> * (Boolean) — true if the element has been hidden, false if the element has not been hidden.

~

**Comment by thomthom** [comment]
Dontcha just love the smell of copypasta?

![image](https://user-images.githubusercontent.com/192418/98934755-d9c97700-24e2-11eb-986a-f8203b36141f.png)

**Comment by DanRathbun** [comment]
NO, not really. **It is evidence of sloppy work and a lack of attention to detail.**

These kind of pastings should have first been pasted into a blank text editor and modified to be correct **before** copying into the target location in the code doc comments. (If the person got interrupted by a meeting or lunch, or whatever, when they came back and saw it was still in a editor by itself, then they'd know it needed to be totally proofed before copying it to where it was intended to be.)
